app/src/main/java/pl/gittobefit/HomeFragment.java

```diff
@@ -54,6 +57,13 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
     @Override
     public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
+        InitiationTrainingDisplayLayoutViewModel model = new ViewModelProvider(requireActivity()).get(InitiationTrainingDisplayLayoutViewModel.class);
+
+
+        if (model.getTrainingWithForms().size() == 0)
+        {
+            model.setTrainingWithForms(TrainingRepository.getInstance(getContext()).getAllTrainingsForUser(""));
+        }
```

Comment:
dlaczego robisz to w homefragment 
pamietaj ze po zalogowaniu w 2 watku sa pobierane treningi z serwera i w user jest specjalny enum mowiacy o tym